README_new.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+At babylon, we firmly believe that **transparency** and **flexibility** are core values that should be present in everything we do, and hiring the best person should reflect that belief.
+
+That's why we in the iOS team have open-sourced our recruitment process, for a fairer assessment process and so that candidates know exactly what to expect, all the way to the [questions we may ask during a face-to-face interview](TODO).
+
+We also aim to be as flexible and inclusive as possible, equally accepting applications from candidates who want to work on-site at our office in London, or who want to work remotely, or who have a limited amount of time to write a demo project, or who have open-source contributions.
+
+### Our expectations
+
+babylon aims to hire only the most qualified people available at any given time. That doesn't mean that a candidate must have several academic degrees or have years and years of experience. We value **potential** equally as much as **knowledge** and **experience**. Above all, we value one's ability to **think independently** and to **engage**. As effective as they may be, we don't want robots.
+
+We also want to hire people who are **reasonable**, **professional**, and **fun** to work with. After all, they are people we will spend a great deal of time interacting with on a daily basis.
```

Comment:
I'd remove the `also`. It's implied. Also good job on the oxford comma.

Reply:
👍